tests/test_classmethods.py

```diff
@@ -15,7 +15,7 @@ def test_from_payload() -> None:
     payload = bytes([i for i in range(Ksuid48.PAYLOAD_LENGTH_IN_BYTES)])
     k = Ksuid48.from_payload(payload)
     assert k.payload == payload
-    assert k.timestamp // 10 == timestamp // 10
+    assert -1 < (k.timestamp - timestamp) < 1
```

Comment:
For `Ksuid48`, shouldn't this be `-0.001 < (k.timestamp - timestamp) < 0.001` or just `abs(k.timestamp - timestamp) < 0.001`?

Reply:
@jalaziz It may cause flaky tests if the precision is high enough and the tests is slow to run in restricted test env (I occurred previously).

Reply:
Ahh my bad, this is `from_payload` where the timestamp is being generated again, didn't realize that, sorry!